CRM/Financial/Form/FinancialBatch.php

```diff
@@ -117,7 +117,7 @@ public function buildQuickForm() {
       $batchStatus = CRM_Core_PseudoConstant::get('CRM_Batch_DAO_Batch', 'status_id');
 
       // unset exported status
-      $exportedStatusId = CRM_Utils_Array::key('Exported', $batchStatus);
+      $exportedStatusId = CRM_Core_PseudoConstant::getKey('CRM_Batch_DAO_Batch', 'status_id', 'Exported');
```

Comment:
As a general practice we should use 'BAO' rather than 'DAO' in case there is an override (you don't need to change the PR just for this but if you are changing the PR BAO is preferred)